plugins/pulp_rpm/plugins/importers/yum/listener.py

```diff
@@ -196,6 +196,7 @@ def download_succeeded(self, report):
                 return
             added_unit = self.sync.add_rpm_unit(self.metadata_files, unit)
```

Comment:
I confirmed the file is checked here https://github.com/pulp/pulp_rpm/blob/master/plugins/pulp_rpm/plugins/importers/yum/existing.py#L129 although it might be an empty file so it's not particularly robust.